docs/docsite/rst/scenario_guides/guide_vmware_rest.rst

```diff
@@ -0,0 +1,20 @@
+.. _vmware_rest_scenarios:
+
+****************************
+vmware.vmware_rest Scenarios
```

Comment:
```suggestion
VMware REST Guide
```

Reply:
Does this title make it easier to understand and differentiate this from the VMware Guide?

Reply:
It's probably better, but I don't have a strong opinion on this.